examples/throughput.rs

```diff
@@ -103,5 +111,5 @@ fn main() {
     let _ = env_logger::init();
     let _ = *BUF; // To non-lazily initialize it.
     bench_tcp(256 << 20);
-    bench_tarpc(256 << 20);
+    bench_tarpc(24 << 20);
```

Comment:
Keep this 256. I'd rather just wait for bincode to fix their performance issues.